x-pack/plugins/security_solution/public/timelines/components/timeline/unified_components/data_table/use_get_event_type_row_classname.ts

```diff
@@ -0,0 +1,24 @@
+import type { TimelineItem } from '@kbn/timelines-plugin/common';
+import { useMemo } from "react";
```

Comment:
Seems like prettier didn't run in this file for some reason 🤔

Reply:
done https://github.com/elastic/kibana/pull/180972/commits/3678fb1dcdbf3aaa0ceea82eb0e83d867164ef7c